modules/benchmark/R/metric.residual.plot.R

```diff
@@ -4,35 +4,31 @@
 ##' @param dat
 ##' 
 ##' @author Betsy Cowdery
-
-metric.residual.plot <- function(dat, var,filename = NA, draw.plot = FALSE){
+metric.residual.plot <- function(dat, var, filename = NA, draw.plot = FALSE) {
   
   library(ggplot2)
-  # 
-  # ind <- intersect(which(!is.na(dat$obvs)),  which(!is.na(dat$model)))
-  # 
-  # dat <- dat[ind,]
+  # ind <- intersect(which(!is.na(dat$obvs)), which(!is.na(dat$model))) dat <- dat[ind,]
   
-  dat$time <- lubridate::year(as.Date(as.character(dat$time), format="%Y"))
+  dat$time <- lubridate::year(as.Date(as.character(dat$time), format = "%Y"))
   dat$diff <- abs(dat$model - dat$obvs)
   dat$zeros <- rep(0, length(time))
   
-  p <- ggplot(data = dat, aes(x=time)) + 
-    geom_path(aes(y=zeros), colour = "#666666", size=2, linetype = 2, lineend = "round") +
-    geom_point(aes(y=diff), size=4,  colour = "#619CFF") + labs(title=var, x= "years", y="abs(model - observation)")
+  p <- ggplot(data = dat, aes(x = time)) 
+  p <- p + geom_path(aes(y = zeros), colour = "#666666", size = 2, linetype = 2, lineend = "round") 
+  p <- p + geom_point(aes(y = diff), size = 4, colour = "#619CFF") 
+  p <- p + labs(title = var, x = "years", y = "abs(model - observation)")
```

Comment:
Personal preference creeps in. It's not my code obviously; can change it back if you'd prefer!